packages/contentful--app-scripts/src/activate/get-activate-args.ts

```diff
@@ -1,16 +1,17 @@
-const chalk = require('chalk');
-const ora = require('ora');
-const { getAppInfo } = require('../get-app-info');
-const { validateArguments } = require('../validate-arguments');
+import chalk from 'chalk';
+import ora from 'ora';
+import { getAppInfo } from '../get-app-info';
+import { validateArguments } from '../validate-arguments';
+import { ActivateOptions, ActivateSettings } from '.';
```

Comment:
By importing from `.` we are creating a circular dependency. It currently works because TypeScript is smart, but I'd avoid doing that.